### PR TITLE
Move from Travis to GitHub actions

### PR DESCRIPTION
**Pull request by ErikSchierboom**
This should hopefully help reduce the time needed to run all tests.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
```
Comment (robkeim) [inline .github/workflows/test.yml]: Is there any reason we're using 16.04 instead of 18.04?

Comment (ErikSchierboom) [reply .github/workflows/test.yml]: No good reason. I've updated!

**Comment by robkeim** [comment]
Thanks @ErikSchierboom 🎉 Fingers crossed this fixes our build issues

**Comment by ErikSchierboom** [comment]
Indeed!

**Pull request merged**